packages/devextreme/js/__internal/grids/grid_core/focus/m_focus.ts

```diff
@@ -205,10 +208,13 @@ export class FocusController extends core.ViewController {
     if (!this.isAutoNavigateToFocusedRow()) {
       this.option('focusedRowIndex', -1);
     }
-    return this._navigateToRow(key);
+    return this._navigateToRow(key, false);
```

Comment:
Question
Why? Isn't it required to focus row when `autoNavigateToFocusedRow` is true?